src/app/modules/submissions/components/submissions-map/submissions-map.component.scss

```diff
@@ -62,6 +49,18 @@
   }
 }
 
+.status{
```

Comment:
Please make sure you format your CSS everywhere.
There is a lot of competition up here, customers will pay attention to all possible details